### PR TITLE
LPS-113582 Deprecate `liferay-notice` AUI component and replaces all occurrences with `openToast`

### PR DESCRIPTION
**Pull request by diegonvs**
I replaced all occurrences of Liferay.Notice with `openToast` utility, since the "Notice" component opens a Toast

### CODE REVIEW

**Comment by diegonvs** [comment]
ci:test:sf

**Comment by diegonvs** [comment]
ci:test:relevant

File: portal-web/docroot/html/common/themes/top_messages.jsp [hunk 1]
```diff
@@ -61,24 +50,26 @@ if (Validator.isNotNull(jspPath) || Validator.isNotNull(message)) {
...
-						duration: 2,
-						top: '0px'
-					},
-				closeText: false,
-				node: '#portalMessageContainer',
-				noticeClass: 'hide <%= cssClass %>',
-				timeout: <%= timeout %>,
-				toggleText: false,
-				useAnimation: <%= useAnimation %>
+	<aui:script>
```
Comment (diegonvs) [inline portal-web/docroot/html/common/themes/top_messages.jsp:35]: Should I use a `<script>` tag instead of `aui:script` taglib?

**Comment by diegonvs** [comment]
```
[exec] > Task :baseline5
     [exec]   PACKAGE_NAME                                       DELTA      CUR_VER    BASE_VER   REC_VER    WARNINGS
     [exec] = ================================================== ========== ========== ========== ========== ==========
     [exec] * com.liferay.portal.kernel.servlet                  MAJOR      8.2.0      8.1.0      9.0.0      VERSION INCREASE REQUIRED
     [exec] 	>   class      com.liferay.portal.kernel.servlet.PortalMessages
     [exec] 		-   field      KEY_ANIMATION
     [exec] 			-   access     final
     [exec] 			-   access     static
     [exec] 			-   constant   animation
     [exec] 			-   return     java.lang.String
     [exec] 	<   interface  com.liferay.portal.kernel.servlet.InactiveRequestHandler
     [exec] 		+   method     isShowInactiveRequestMessage()
     [exec] 			+   return     boolean
     [exec] 	-   version    8.1.0
     [exec] 	+   version    8.2.0
     [exec] [Baseline Report] Mode: diff (persisted)
     [exec] Semantic versioning is incorrect while checking /Users/diegonascimento/workspace/liferay-portal/portal-kernel/portal-kernel.jar against /Users/diegonascimento/workspace/liferay-portal/.gradle/caches/modules-2/files-2.1/com.liferay.portal/com.liferay.portal.kernel/5.4.0/9febcd2f258b6fcd2a5c80f62a637d0921a8c607/com.liferay.portal.kernel-5.4.0.jar
     [exec]
     [exec] > Task :baseline6
     [exec]   PACKAGE_NAME                                       DELTA      CUR_VER    BASE_VER   REC_VER    WARNINGS
     [exec] = ================================================== ========== ========== ========== ========== ==========
     [exec] * com.liferay.portal.kernel.servlet                  MAJOR      8.2.0      8.1.0      9.0.0      VERSION INCREASE REQUIRED
     [exec] 	>   class      com.liferay.portal.kernel.servlet.PortalMessages
     [exec] 		-   field      KEY_ANIMATION
     [exec] 			-   access     final
     [exec] 			-   access     static
     [exec] 			-   constant   animation
     [exec] 			-   return     java.lang.String
     [exec] 	<   interface  com.liferay.portal.kernel.servlet.InactiveRequestHandler
     [exec] 		+   method     isShowInactiveRequestMessage()
     [exec] 			+   return     boolean
     [exec] 	-   version    8.1.0
     [exec] 	+   version    8.2.0
     [exec] [Baseline Report] Mode: diff (persisted)
     [exec] Semantic versioning is incorrect while checking /Users/diegonascimento/workspace/liferay-portal/portal-kernel/portal-kernel.jar against /Users/diegonascimento/workspace/liferay-portal/.gradle/caches/modules-2/files-2.1/com.liferay.portal/com.liferay.portal.kernel/6.10.1/18155b3b72ad4c466cf7d3ab72e84fd891e124c6/com.liferay.portal.kernel-6.10.1.jar
     [exec]
     [exec] > Task :baseline7
     [exec]   PACKAGE_NAME                                       DELTA      CUR_VER    BASE_VER   REC_VER    WARNINGS
     [exec] = ================================================== ========== ========== ========== ========== ==========
     [exec] * com.liferay.portal.kernel.servlet                  MAJOR      8.2.0      8.2.0      9.0.0      VERSION INCREASE REQUIRED
     [exec] 	>   class      com.liferay.portal.kernel.servlet.PortalMessages
     [exec] 		-   field      KEY_ANIMATION
     [exec] 			-   access     final
     [exec] 			-   access     static
     [exec] 			-   constant   animation
     [exec] 			-   return     java.lang.String
     [exec] [Baseline Warning] Bundle Version Change Recommended: 8.0.0
     [exec] [Baseline Report] Mode: diff (persisted)
     [exec] Semantic versioning is incorrect while checking /Users/diegonascimento/workspace/liferay-portal/portal-kernel/portal-kernel.jar against /Users/diegonascimento/workspace/liferay-portal/.gradle/caches/modules-2/files-2.1/com.liferay.portal/com.liferay.portal.kernel/7.3.0/8eeb4aa93678aa973e5a4dd0a480e27f74d3ae06/com.liferay.portal.kernel-7.3.0.jar
     [exec]
     [exec] > Task :baseline4
     [exec]   PACKAGE_NAME                                       DELTA      CUR_VER    BASE_VER   REC_VER    WARNINGS
     [exec] = ================================================== ========== ========== ========== ========== ==========
     [exec] * com.liferay.portal.kernel.servlet                  MAJOR      8.2.0      8.2.0      9.0.0      VERSION INCREASE REQUIRED
     [exec] 	>   class      com.liferay.portal.kernel.servlet.PortalMessages
     [exec] 		-   field      KEY_ANIMATION
     [exec] 			-   access     final
     [exec] 			-   access     static
     [exec] 			-   constant   animation
     [exec] 			-   return     java.lang.String
     [exec] [Baseline Report] Mode: diff (persisted)
     [exec] Semantic versioning is incorrect while checking /Users/diegonascimento/workspace/liferay-portal/portal-kernel/portal-kernel.jar against /Users/diegonascimento/workspace/liferay-portal/.gradle/caches/modules-2/files-2.1/com.liferay.portal/com.liferay.portal.kernel/4.57.1/aa14da5a63817b8dd7b43f46d583d9824ecdb723/com.liferay.portal.kernel-4.57.1.jar
     [exec]
     [exec] > Task :baseline3
     [exec] > Task :baseline
     [exec]
     [exec] BUILD SUCCESSFUL in 9s
     [exec] 6 actionable tasks: 6 executed
     [exec]

init-deploy:

deploy:
     [copy] Copying 1 file to /Users/diegonascimento/workspace/bundles/tomcat-9.0.33/lib/ext
     [copy] Copying 12 files to /Users/diegonascimento/workspace/bundles/tomcat-9.0.33/webapps/ROOT/WEB-INF/classes
     [copy] Copying 1 file to /Users/diegonascimento/workspace/bundles/tomcat-9.0.33/webapps/ROOT/WEB-INF/lib

BUILD SUCCESSFUL
Total time: 1 minute 6 seconds
```

How can I bump this version?

**Comment by diegonvs** [comment]
ci:test:sf

**Comment by diegonvs** [comment]
ci:test:relevant

**Comment by wincent** [comment]
@diegonvs As a general rule, would you mind sending this to me as a PR against [my fork](https://github.com/wincent/liferay-portal/compare)? That way I can apply small fixes (if necessary) and forward from there.

File: portal-web/docroot/html/common/themes/top_messages.jsp (hunk 1 above)
Comment (wincent) [reply portal-web/docroot/html/common/themes/top_messages.jsp:35]: If there is no dependency on AUI, there is no benefit to using `aui:script`, AFAIK. At least, that was my conclusion when I looked at the difference in the generated output; eg. in [this commit](https://github.com/brianchandotcom/liferay-portal/pull/88591/commits/44eb433d6a2c44c40c71fa24ba919d1db6dd33ad), I wrote:

> In terms of generated output, the only difference is that the `aui:script` tag makes an IIFE that sets up some local variables that the scripts don't use:
>
> ```
> var $ = AUI.$;var _ = AUI._;
> ```

So, that's at least how I understand things work.

**Comment by wincent** [comment]
Forwarded: https://github.com/wincent/liferay-portal/pull/297

**Comment by wincent** [comment]
ci:close

**Pull request closed without merging**